framework/src/utils/Conversion.C

```diff
@@ -412,6 +412,8 @@ stringify(FEFamily f)
       return "NEDELEC_ONE";
     case 43:
       return "MONOMIAL_VEC";
+    case 44:
+      return "RAVIART_THOMAS";
```

Comment:
Not sure why this switch exists - couldn't the whole thing have been replaced with a `return Utility::enum_to_string(f);`?

Reply:
Done.

We could also remove the quadrature-to-enum map and function in MOOSE in favour of libMesh's but that would require a minor change to the latter which is missing the CONICAL and MONOMIAL quadrature rules in its string to enum map for some reason. Are you up for it? Maybe in another PR. ^^

https://github.com/idaholab/moose/blob/0e39829646255a0ff49ba89a2de43d07b07f8aab/framework/src/utils/Conversion.C#L35-L49

Reply:
Yeah, I'd say a separate PR is a good idea there, especially if it'll have to wait on a libMesh update.  It might be that the `CONICAL` rules were left out of that translation on purpose, on the theory that they're mostly just useful as fallbacks for `QGauss`, but omitting an entry for `MONOMIAL` had to be an oversight, and IMHO we should have translations for every option even if we expect some to go unused.